<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameDetailsActivity</class>
 <widget class="QWidget" name="GameDetailsActivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="mp_layout">
     <item>
      <widget class="QPushButton" name="mp_btn_close">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
       <property name="icon">
        <iconset theme="oplpct-back"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="OplPcTools::UI::ClickableLabel" name="mp_label_title">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mp_btn_rename_game">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip">
        <string>Rename...</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="oplpct-rename"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="mp_list_arts">
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="mp_action_change_art">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/document-edit</normaloff>:/icons/document-edit</iconset>
   </property>
   <property name="text">
    <string>Change Picture...</string>
   </property>
  </action>
  <action name="mp_action_delete_art">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/edit-delete</normaloff>:/icons/edit-delete</iconset>
   </property>
   <property name="text">
    <string>Delete Picture</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OplPcTools::UI::ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">OplPcTools/UI/ClickableLabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mp_btn_close</tabstop>
  <tabstop>mp_btn_rename_game</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mp_btn_rename_game</sender>
   <signal>clicked()</signal>
   <receiver>GameDetailsActivity</receiver>
   <slot>renameGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_list_arts</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>GameDetailsActivity</receiver>
   <slot>changeGameArt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>renameGame()</slot>
  <slot>changeGameArt()</slot>
 </slots>
</ui>
